how-tos/how_to_guide_parking_lot.md

```diff
@@ -0,0 +1,37 @@
+# How-To Guides Parking Lot
+
+This is a document to capture and organize How-To Guides that do not,
+yet, have a place in the Cookbook.  I've pulled examples from the
+[Hackday Google Doc](https://docs.google.com/document/d/1fzT-iSFlWZLS38eoPmFseljyMKDQIAc-24qH6QpnRCc/edit#)
+
+Not all of these how-tos may be required.  Some may be redundant.
+Some may be too complicated and could be broken into several How-Tos.
+Feel free to add comments, links, notes, etc under each suggested
+How-To.  Also suggest changes to the wording.  Following the Divio
+documentation guide, I've tried to name how-tos with descriptive names
+that start "How do I... ?". 
+
+_We need a page to explain terminology; collections, granules, ..._
+
+How do I use Harmony to reproject a data granule?
+
+How do I search the USGS Landsat Cloud?
+How do I access data in the USGS Landsat Cloud?
+
+How do I find OpenDAP datasets?  
+How do I open an OpenDAP granule in the cloud?  Include Authentication?  
+
+How do I open/access an OpenDap granule from my local machine?  
+
+How do I open a Zarr granule in the cloud?
+How do I open a COG (Cloud-Optimized-GeoTIFF) in the cloud?
+
+How do I use Kerchunk to reformat granules?  _needs refining_
+
+How do I use CMR STAC to access data in the cloud?
+
+How do I use Matlab in the cloud?
+
+How do I add a python package to a 2i2c environment?
```

Comment:
"How do I work in a shared cloud environment?"  I think this needs refining.  Are you thinking "How do I access a 2i2c" or some other cloud instance.

I've added the "...reproduce on my local machine" to the parking lot.

Reply:
I suggest we keep the parking lot as an evolving doc.

Reply:
@andypbarrett Sure, it could certainly be refined. I was thinking about some sort of category for python environments that could include adding a python package to 2i2c and also how to reproduce an environment. But they don't necessarily need to be grouped. Thanks for adding the latter how-to. Agreed on keeping that parking lot as an evolving doc.